Added Studio docs for email sign up and custom Gitlab server support (#3419)
iterative/dvc.org: content/docs/studio/user-guide/account-management.md

profile page will open. This page has multiple sections that are described
below.

## Profile

Here, you can see your name and profile picture. If you signed up with a

> GitHub.com, GitLab.com or Bitbucket.org account, these details are fetched from

Reviewer: OK all the .coms and .orgs start to get repetitive after a bit. I'd ✂️ them (minor).

Author: See previous comment for why I added them.

your connected Git hosting account.

You can edit your name.

## Account

Here, you can see your username, password and email addresses. If you signed up
with a GitHub.com, GitLab.com or Bitbucket.org account, the username and email
address are fetched from your connected Git hosting account.

You can update your username and password.

**Managing email addresses:**

You can add multiple email addresses to a single Studio account. You can login

> to the account with any of your verified email addresses.

Reviewer: > You can add multiple email addresses to a single Studio account. You can login to the account with any of your verified email addresses.

May I ask why multiple addresses is allowed?

And furthermore why allowing login with any of them? And is that true even if the signup was via GH/GL/BB? (unclear from this doc)

Author: Multiple email addresses are allowed, just like in GitHub. The primary one can't be deleted; it's the one that gets account notifications. You could change your email address for a Studio account by adding a new one and setting that as primary and then deleting the old (non-primary) one.

If they are all my email addresses, they why not let me log in with any of them? Yes, it's true regardless of the signup method, as long as you have set a password in your account - have added this to the doc.

Reviewer: > Multiple email addresses are allowed, just like in GitHub

Yes but why? Is there a use case? I'm just curious -- sounds like possibly unnecessary complication.

Author: Someone could use either of their email addresses to login (if they have multiple - such as for work and personal). It also lets them change the email address for their Studio account by adding the new one and deleting the old one (although for this, we could have added a `Change email` functionality). Also, I don't remember, but I think there was another (more technical) reason for this that applied to email signups. I can ask in the team if necessary.

Reviewer 2: It's possible to lose access to email accounts for some reason and keeping multiple email addresses recorded in services (if possible) is usually a better approach. @jorgeorpinel

One of your email addresses must be designated as primary. This is the address
to which Studio will send all your account notification emails.

You can change your primary email address by clicking on the `Primary` button
next to the email address which you want to designate as primary.

You can delete your non-primary email addresses.

## Git integrations

In this section, you can,

- Connect to GitHub.com, GitLab.com or Bitbucket.org.

>   When you connect to a Git provider, you may be prompted to grant access to

Reviewer: provider -> hosting?

(Consistent terminology vs other pages)

Author: Changed it to `Git hosting provider`

Reviewer: Thanks

>   your account to Iterative Studio. Please grant the required access to enable

Reviewer: > to grant access to your account to Iterative Studio

-> to grant Iterative Studio access to your account

Author: Done.

>   Iterative Studio to connect to your Git repositories.

Reviewer: The last sentence is redundant, I think.

Author: Removed.

>   If you signed up to use Studio with an email address, you will not have any of
>   the Git connections when you first sign in to Studio, and you can set up the
>   required Git connections in this section.

Reviewer: More copy edits:
```suggestion
  If you signed up to use Studio with an email address, you will not have any
  Git connections by default. You can set them in this section.
```

Author: Changed. Minor edit -> Changed `You can set them in this section.` to `You can set them up in this section.`

  Note that **connections to self-hosted GitLab servers** are not managed in
  this section. If you want to connect to a self-hosted GitLab server, you

>   should create a team and set up the GitLab server connection in the team
>   settings. For more details, refer
>   [here](/doc/studio/user-guide/connect-custom-gitlab-server).

Reviewer: ```suggestion
  should create a team and
  [set up the GitLab server connection](/doc/studio/user-guide/connect-custom-gitlab-server)
  in the team settings.
```
> Needs wrapping

> - Disconnect from your GitHub.com, GitLab.com or Bitbucket.org accounts.

Reviewer: ```suggestion
- Disconnect from your GitHub, GitLab or Bitbucket account.

```

Author: There's a reason the `.com/org`s were addded (see previous comments). But I guess the user will understand here, so I've removed then as you suggested. But I didn't change `accounts` to `account`, coz the user can disconnect from multiple Git accounts (each on a different provider).

profile page will open. This page has multiple sections that are described
below.

## Profile

Here, you can see your name and profile picture. If you signed up with a
GitHub.com, GitLab.com or Bitbucket.org account, these details are fetched from

> your connected Git account.

Author: How about `Git cloud`?

> your connected Git account.

Reviewer: I'm not sure these services are considered clouds. I wouldn't use that term here.

> your connected Git account.

Author: So you suggest keeping `your connected Git account`?

You can edit your name.

## Account

Here, you can see your username, password and email addresses. If you signed up
with a GitHub.com, GitLab.com or Bitbucket.org account, the username and email

> address are fetched from your connected Git account.

Author: How about `Git cloud`?

> address are fetched from your connected Git account.

Reviewer: "Git hosting account" ?

> address are fetched from your connected Git account.

Author: Done.

You can update your username and password.

**Managing email addresses:**

You can add multiple email addresses to a single Studio account. For each new
email address that you add, you will receive an email with a link to verify the

> email address.

Author: I changed the content a bit. How is this: `You can add multiple email addresses to a single Studio account. You can login to the account with any of your verified email addresses.`?

One of your email addresses must be designated as primary. This is the address
to which Studio will send all your account notification emails.

You can change your primary email address by clicking on the `Primary` button
next to the email address which you want to designate as primary.

You can delete your non-primary email addresses.

## Git integrations

In this section, you can,

> - Connect to the following Git providers: GitHub.com, GitLab.com, Bitbucket.org.

Author: Done

>   When you connect to a Git provider, you may be prompted to Grant access to

Author: Done
